### DISCUSSION TITLE
Feature request : Command to reset the PulseTime countdown

### DISCUSSION THREAD

**Discussion by jburdy** [arendst/Tasmota#13785] [Q&A] [answered]
Hello,

It would be very useful to have a command to reset the PulseTime countdown. To occasionally extend its duration. Without having to know its initial value (because yes by setting it again it works, but I will not put the value in the command).

Thanks

**Comment by sfromis** [comment, marked as answer]
Tasmota already has what you ask for. The way PulseTime works, you just should not set it again (unless you want a different value, of course), as it is saved persistently. It is used when you power on the output, a new countdown starts each time. You can extend the duration by powering on again.

**Reply by jburdy** [reply]
So simple I hadn't considered it!!! I only had TOGGLE easily accessible (HAss). Thanks!
